sbc_email/models/import_letters_history.py

```diff
@@ -151,6 +151,81 @@ def button_import(self):
 
             return super(ImportLettersHistory, self).button_import()
 
+    @api.multi
+    def button_save(self):
+        """
+        When saving the import_line as correspondences, move pdf file in the
+        done folder on the NAS and remove image attachment (Web letter case)
+        """
+        for letter in self:
+            if letter.config_id.name == "Web Letter":
+                for line in letter.import_line_ids:
+                    # build part of filename, corresponding to this web letter
+                    # remove part after '-' caracter (including) corresponding
+                    # to the page number
+                    part_filename = (line.letter_image.name[:-4]).split('-')[0]
+
+                    share_nas = self.env.ref('sbc_email.share_on_nas').value
+
+                    smb_conn = self._get_smb_connection()
+                    if smb_conn and smb_conn.connect(
+                            SmbConfig.smb_ip, SmbConfig.smb_port):
+                        imported_letter_path = self.env.ref(
+                            'sbc_email.scan_letter_imported').value
+                        listPaths = smb_conn.listPath(
+                            share_nas,
+                            imported_letter_path)
+
+                        # loop in import folder and find pdf corresponding to
+                        # this web letter and eventual attached image
+                        image_ext = None
+                        file_to_save = False
+                        for sharedFile in listPaths:
+                            ext = sharedFile.filename[-4:]
+                            if part_filename == sharedFile.filename[:-4]:
+                                if ext == '.pdf':
+                                    file_to_save = True
```

Comment:
Not necessary in this case, the test made before is similary to test made in is_in_list_letter function